Sources/Networking/Core/Upload/UploadAPIManaging.swift

```diff
@@ -63,13 +64,13 @@ public protocol UploadAPIManaging {
     /// i.e., `UploadTask.State.error` is non-nil. In such case, you can call `retry(taskId:)` to re-activate the stream for the specified `uploadTaskId`.
     /// - Parameter uploadTaskId: The identifier of the task to observe.
     /// - Returns: An asynchronous stream of upload state. If there is no such upload task the return stream finishes immediately.
-    func stateStream(for uploadTaskId: UploadTask.ID) async -> StateStream
+    func stateStream(for uploadTaskId: UploadTask.ID) -> StateStream
```

Comment:
Similar question like for other class. Why do we change async to sync method and vice versa? Because of context where does it run?

Reply:
I think it makes sense for methods that don't necessarily need to be async to let them be sync and this one fulfils that criteria. Don't you agree?